industry_restaurant/demo/website_view.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <template id="industry_restaurant_appointment" inherit_id="appointment.appointment_meeting_details">
+        <xpath expr="//small[hasclass('text-uppercase')]" position="replace">
+            <small class="text-uppercase text-muted">Table Booking Details</small>
```

Comment:
Is it still translatable from the frontend?

Reply:
No it is not translatable

Reply:
Then I'm afraid we'll need to redefine the whole view in order to keep the page translatable

Reply:
EDIT: no, page is still translatable